flatpak_builder_lint/checks/desktop.py

```diff
@@ -99,19 +109,21 @@ def _validate(self, path: str, appid: str) -> None:
                 self.errors.add("desktop-file-exec-key-absent")
                 pass
 
-            try:
-                hidden = d["Desktop Entry"]["Hidden"]
-                if hidden == "true":
-                    self.errors.add("desktop-file-is-hidden")
-            except KeyError:
-                pass
-
-            try:
-                nodisplay = d["Desktop Entry"]["NoDisplay"]
-                if nodisplay == "true":
-                    self.errors.add("desktop-file-is-hidden")
-            except KeyError:
-                pass
+            if not is_console:
+
+                try:
+                    hidden = d["Desktop Entry"]["Hidden"]
```

Comment:
this is should stay even for console applications.

> Hidden should have been called Deleted. It means the user deleted (at their level) something that was present (at an upper level, e.g. in the system dirs). It's strictly equivalent to the .desktop file not existing at all, as far as that user is concerned. This can also be used to "uninstall" existing files (e.g. due to a renaming) - by letting make install install a file with Hidden=true in it.

Reply:
Hmm yea, I'll change it

Reply:
It's so used interchangeably with NoDisplay that a bunch of console applications will probably fail.